src/workflow.ts

```diff
@@ -89,7 +89,7 @@ export class WorkflowContextImpl extends OperonContextImpl implements WorkflowCo
       },
       parentCtx?.span,
     );
-    super(workflowName, span, operon.logger, parentCtx);
+    super(workflowName, span, operon.logger, operon.config.applicationVersion, parentCtx);
```

Comment:
All context constructors pass the `applicationVersion` to the parent `OperonContext`